data/virt_autotest/kubevirt_tests/core-tests.conf

```diff
@@ -29,3 +29,6 @@ GINKGO_FOCUS=test_id:1532
 
 [replicaset_test]
 GINKGO_FOCUS=test_id:4121
+
+[hotplug_test]
```

Comment:
Would be better to put it under `migration_test`. This is the perspective that the test is selected for.

Reply:
Besides, from the verification job, among uploaded logs, there seems no report for this new added category. Do I overlook it anywhere?

```
[0_console_test.html](http://10.67.129.96/tests/3173/file/0_console_test.html)
 [1_container_disk_test.html](http://10.67.129.96/tests/3173/file/1_container_disk_test.html)
 [2_datavolume_test.html](http://10.67.129.96/tests/3173/file/2_datavolume_test.html)
 [3_expose_test.html](http://10.67.129.96/tests/3173/file/3_expose_test.html)
 [4_replicaset_test.html](http://10.67.129.96/tests/3173/file/4_replicaset_test.html)
 [5_storage_test.html](http://10.67.129.96/tests/3173/file/5_storage_test.html)
 [6_vmi_lifecycle_test.html](http://10.67.129.96/tests/3173/file/6_vmi_lifecycle_test.html)
 [7_vnc_test.html](http://10.67.129.96/tests/3173/file/7_vnc_test.html)
```

Reply:
> Would be better to put it under `migration_test`. This is the perspective that the test is selected for.

Actually, it's belong to hotplug test suites.
- [sig-storage] Hotplug [storage-req] VMI migration should allow live migration with attached hotplug volumes containerDisk VMI
- [sig-storage] Hotplug [storage-req] VMI migration should allow live migration with attached hotplug volumes persistent disk VMI

Reply:
Yes, noticed before giving the comment ;-)

Reply:
> Besides, from the verification job, among uploaded logs, there seems no report for this new added category. Do I overlook it anywhere?
> 
> ```
> [0_console_test.html](http://10.67.129.96/tests/3173/file/0_console_test.html)
>  [1_container_disk_test.html](http://10.67.129.96/tests/3173/file/1_container_disk_test.html)
>  [2_datavolume_test.html](http://10.67.129.96/tests/3173/file/2_datavolume_test.html)
>  [3_expose_test.html](http://10.67.129.96/tests/3173/file/3_expose_test.html)
>  [4_replicaset_test.html](http://10.67.129.96/tests/3173/file/4_replicaset_test.html)
>  [5_storage_test.html](http://10.67.129.96/tests/3173/file/5_storage_test.html)
>  [6_vmi_lifecycle_test.html](http://10.67.129.96/tests/3173/file/6_vmi_lifecycle_test.html)
>  [7_vnc_test.html](http://10.67.129.96/tests/3173/file/7_vnc_test.html)
> ```

Code fixed. Thanks for pointing out!

Reply:
> Yes, noticed before giving the comment ;-)

Nan, I guess some misunderstanding on this? I mean I noticed it's test from hotplugging file, but given that the suggestion from Vasily for adding the test is from migration perspective, so still be good to put it under [migration test]. Yes, a minor change, but worthwhile since it is top message from report.

Reply:
OK, updated.